src/sounds.cpp

```diff
@@ -160,9 +161,48 @@ static std::vector<centroid> cluster_sounds( std::vector<std::pair<tripoint, int
     return sound_clusters;
 }
 
+int get_signal_for_hordes(int vol)
+{
+    int min_vol_for_hordes_cap = 61;
+    int per_level_down_vol_reduce_coef_for_hordes = 2;
+    int min_vol_for_hordes = 61;
+    int max_vol_for_hordes = 200;
+    int min_sig_power_for_hordes = 4;
+    int max_sig_power_for_hordes = 8;
```

Comment:
Those names are giant, which makes it pretty hard to read the function.
It would be better if you shortened them to shortest readable name and added a comment if it's ambiguous.

For example `min_signal`, `max_vol`, `vol_signal_coeff` etc.